identity_cache.gemspec

```
@@ -22,7 +22,9 @@ Gem::Specification.new do |gem|
                       "cannot be found in Memcached."
   gem.homepage      = "https://github.com/Shopify/identity_cache"
 
-  gem.files         = `git ls-files`.split($\)
+  gem.files         = Dir.chdir(File.expand_path("..", __FILE__)) do
+    %x(git ls-files -z).split("\x0").reject { |f| f.match(%r{^(test|spec|features)/}) }
```

There aren't top-level `spec` or `features` directories, so why bother adding those?

```suggestion
    %x(git ls-files -z).split("\x0").reject { |f| f.match(%r{^test/}) }
```

---

Same pattern as the one used in `gem.test_files`. I've updated both to be less generic. 👍